### ISSUE TITLE
Failed at stage 3 with master branch

### ISSUE THREAD

**Issue by pieman64** [andreherberth/pockethold#30]
@andreherberth like the look of Pockethold. Tried your master branch and clicked through stages 1, 2 and 3 but the install failed.

flarumtemp and composer (inside vendor) are both empty.

Server details:

```
Apache Version 2.4.29
PHP Version  5.6.32
MySQL Version  10.0.31-MariaDB-cll-lve
Architecture  x86_64
Operating System  linux
```
Presumably /FlarumTest is ok and it doesn't need to install in the root directory?
I currently have a 302 temporary redirect on the root of the domain but  not the directory for Flarum.

I tried a second time and stage 1 does create the temp directory with res, src, bin and vendor directories and vendor has the composer and symfony subdirectories etc. Stage 2 seems to quit very early with something like "1 of 91" flashing up on screen before moving onto stage 3.

install.log has:
```
Mon Nov 13 22:26:44: Status:  prepare
Mon Nov 13 22:26:53: Status:  Composer is Unpacked
Mon Nov 13 22:26:59: Status:  composer
Mon Nov 13 22:30:02: Status:  waiting1
Mon Nov 13 22:30:02: Status:  /home/domain/public_html/test/temp/
Mon Nov 13 22:30:02: Status:  composer.log is currently 1 long
Mon Nov 13 22:30:12: Status:  cleanup1
```

OK think I have found the problem. composer.log shows:
```
Installing flarum/flarum (v0.1.0-beta.7)
  - Installing flarum/flarum (v0.1.0-beta.7): Downloading (100%)
    <warning>Failed to download flarum/flarum from dist: The Process class relies on proc_open, which is not available on your PHP installation.</warning>
    <warning>Now trying to download from source</warning>
  - Installing flarum/flarum (v0.1.0-beta.7): 
                                                                                           
  [Symfony\Component\Process\Exception\RuntimeException]                                   
  The Process class relies on proc_open, which is not available on your PHP installation.  
                                                                                           

create-project [-s|--stability STABILITY] [--prefer-source] [--prefer-dist] [--repository REPOSITORY] [--repository-url REPOSITORY-URL] [--dev] [--no-dev] [--no-custom-installers] [--no-scripts] [--no-progress] [--no-secure-http] [--keep-vcs] [--no-install] [--ignore-platform-reqs] [--] [<package>] [<directory>] [<version>]
```

Any way around the proc_open if the host will not enable it?

**Comment by andreherberth** [comment]
Hi @pieman64 , 

Alas, I have not found a way, baring offering a precompiled version, to counter proc_open not available. 
Link: https://github.com/andreherberth/pockethold/files/1430027/flarumnew.zip 

Alas, that compile have bazaar included. 
I am not sure if bazaar would work without proc_open or not, but I assume it won't. 

You could try though, if it works, I need to scan their code properly to see what they do differently when they use composer to install the plugins.  

Sorry for the inconvenience :/

**Comment by andreherberth** [comment]
Will in theory be fixed by #31
